x/validator/types/rules.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import dclauthtypes "github.com/zigbee-alliance/distributed-compliance-ledger/x/dclauth/types"
+
+// TODO: 1. Move it to separate module  	2. Make it configurable		3. Save into store.
+var (
+	DisableValidatorPercent = 0.66
+	DisableValidatorRole    = []dclauthtypes.AccountRole{dclauthtypes.Trustee, dclauthtypes.NodeAdmin}
```

Comment:
It makes sense to have 2 separate constants `VoteForDisableValidatorRole` and `DisableValidatorRole` (`StraightDisableValidatorRole`?). So there would be no necessity to use an explicit role from `dclauthtypes` where only one role is allowed.

Reply:
added